adapters/adview/adview.go

```diff
@@ -0,0 +1,167 @@
+package adview
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"text/template"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type adViewAdapter struct {
+	EndpointTemplate template.Template
```

Comment:
The type needs to be `*template.Template`. This was changed recently. Due to this, the code in this PR does not compile.

Reply:
roger.